src/containers/DidDocumentView/DidDocumentView.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { Contact } from '../../types/Contact'
+import './DidDocumentView.scss'
+import Code from '../../components/Code/Code'
+
+type Props = {
+  didDocument: object
+  children: any
+}
+
+const DidDocumentView = (props: Props) => {
+  const { didDocument } = props
+  return (
+    <div className="attributes">
+      <div>
```

Comment:
Question:
Do you need this div?

Reply:
No, it isn't needed. It has been removed.